.github/workflows/codeql-analysis.yml

```diff
@@ -36,7 +27,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'cpp', 'python' ]
+        language: ['cpp', 'python']
```

Comment:
Not sure, but cpp is the correct language specifier here, instead of c?

Reply:
It uses `cpp` for both of them:

https://github.com/github/codeql-action/blob/fd4dc5bf31bec2a626bf9a89113359953a1fba43/src/languages.ts#L5-L23

Reply:
Thanks.